tools/make_class_power.py

```diff
@@ -157,6 +163,10 @@ def make_class_power(paramfile, external_pk = None, extraz=None):
     powerparams = {'output': 'dTk vTk mPk', 'P_k_max_h/Mpc' : maxk, "z_max_pk" : 1+np.max(outputs),'z_pk': outputs, 'extra metric transfer functions': 'y'}
     pre_params.update(powerparams)
 
+    if verbose:
+        verb_params = {'input_verbose': 1, 'background_verbose': 1, 'thermodynamics_verbose': 1, 'perturbations_verbose': 1, 'transfer_verbose': 1, 'primordial_verbose': 1, 'spectra_verbose': 1, 'nonlinear_verbose': 1, 'lensing_verbose': 1, 'output_verbose': 1}
+        pre_params.update(verb_params)
+
     #Specify an external primordial power spectrum
     if external_pk is not None:
```

Comment:
This is fine, but I think it would be nicer to bypass the pk code and directly multiply transfer to the provided pkprim instead? It feels cleaner than doing the command; I can keep it on my todo list.

Reply:
That's a good idea, would simplify the transfer function code a bit. We would need to set whichever class setting it is to output the primordial power.